takwimu/templates/profile/profile_detail_takwimu.html

```diff
@@ -80,50 +96,50 @@ <h2 class="header-for-columns">Crop Production in Tonnes </h2>
-              Methods</h2>
-            <div class="column-full">
-              <div id="chart-grouped_column-health_centers-prevention_methods_dist"
-                   data-stat-type="scaled-percentage"
-                   data-chart-title=""></div>
-            </div>
-          </section>
-        </div>
-      {% endif %}
-    </article>
-  {% endif %}
+{#  {% if not health_centers.is_missing %}#}
+{#    <article id="health_centers_distribution" class="clearfix">#}
+{#      <header class="section-contents">#}
+{#        <h1>Health centers distribution</h1>#}
+{#      </header>#}
+{#      {% if not health_centers.health_centers_dist.is_missing or not health_centers.hiv_health_centers_dist.is_missing or not health_centers.health_centers_ownership_dist.is_missing or not health_centers.health_centers_dist.is_missing %}#}
+{#        <div class="section-container">#}
+{#          <section class="clearfix stat-row">#}
+{#            {% if not health_centers.health_centers_dist.is_missing %}#}
+{#              <div class="column-third">#}
+{#                {% include 'profile/_blocks/_stat_list.html' with stat=health_centers.total_health_centers_dist stat_type='number' %}#}
+{#              </div>#}
+{#            {% endif %}#}
+{#            {% if not health_centers.hiv_health_centers_dist.is_missing %}#}
+{#              <div class="column-third">#}
+{#                {% include 'profile/_blocks/_stat_list.html' with stat=health_centers.total_hiv_health_centers_dist stat_type='number' %}#}
+{#              </div>#}
+{#            {% endif %}#}
+{#            {% if not health_centers.health_centers_ownership_dist.is_missing %}#}
+{#              <div class="column-third" id="chart-pie-health_centers-health_centers_ownership_dist" data-initial-sort="-value" data-stat-type="scaled-percentage" data-initial-sort="-value" data-chart-title="Ownership"></div>#}
+{#            {% endif %}#}
+{#            {% if not health_centers.health_centers_dist.is_missing %}#}
+{#              <div class="column-full">#}
+{#                <div id="chart-histogram-health_centers-health_centers_dist" data-stat-type="scaled-percentage" data-chart-title="Number of health centers by types (2014)"></div>#}
+{#              </div>#}
+{#            {% endif %}#}
+{#          </section>#}
+{#        </div>#}
+{#      {% endif %}#}
+{#      {% if not health_centers.prevention_methods_dist.is_missing %}#}
+{#        <div class="section-container">#}
+{#          <section class="clearfix stat-row">#}
+{#            <h2 class="header-for-columns">Knowledge of HIV Prevention#}
+{#              Methods</h2>#}
+{#            <div class="column-full">#}
+{#              <div id="chart-grouped_column-health_centers-prevention_methods_dist"#}
+{#                   data-stat-type="scaled-percentage"#}
+{#                   data-chart-title=""></div>#}
+{#            </div>#}
+{#          </section>#}
+{#        </div>#}
+{#      {% endif %}#}
+{#    </article>#}
+{#  {% endif %}#}
```

Comment:
Again, if it needs to be removed, remove it and explain it in the commit message.

Reply:
@kilemensi working on fixing the health centers, they had an issue that was causing everything to fail